src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -626,26 +623,68 @@ private void InitializeOutputPane()
             cancellationToken.ThrowIfCancellationRequested();
             OutputToOutputWindow($"Getting project information - done (took {DateTimeOffset.UtcNow - start})");
 
+            ForceLoadProjectsWhoseDesignTimeBuildFailed(projectInfos);
```

Comment:
Does this force load dependent projects? If B depends on A, and B failed for some reason, will this trigger a load on A which _didn't_ fail but CreateDeferredProjects might do something bad with now?